### PR TITLE
Improving imports and adding CI workflow

### PR DESCRIPTION
**Pull request by PBundyra**

### CODE REVIEW

File: airflow_dvc/plugin/plugin.py [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
 from flask_appbuilder import BaseView as AppBuilderBaseView
 from flask_appbuilder import expose
+from .git_url_parser import parse as parse_git_url
 
-from airflow_dvc import (DVCCommit, DVCDownloadOperator, DVCHook,
-                         DVCUpdateOperator, DVCUpdateSensor)
+import sys
 
-from .git_url_parser import parse as parse_git_url
+sys.path.append("..")
```
Comment (styczynski) [inline airflow_dvc/plugin/plugin.py:14]: What happened here? Path manipulation is wrong. Please do it correctly and check if it's really needed?

**Review by styczynski — CHANGES_REQUESTED** [review]
Before this PR can be merged please fix all CI errors.

[Logs are available here](https://github.com/covid-genomics/airflow-dvc/pull/5/checks?check_run_id=3310607880)

Screenshot 2021-08-12 at 12 40 12

**Review by styczynski — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+name: Build and test
+
+on:
+  push:
+    branches: [ master, feature/ci-workflow ]
```
Comment (styczynski) [inline .github/workflows/ci.yml:5]: We will also need to remember to change that, but workflow looks fine

Comment (PBundyra) [reply .github/workflows/ci.yml:5]: I have already noticed that but please note that the workflow doesnt publish the package

Comment (PBundyra) [reply .github/workflows/ci.yml:5]: Actually I will change it now

**Pull request merged**